mlir/test/CAPI/mixr_full.c

```diff
@@ -229,7 +229,7 @@ static bool constructAndTraverseIr(MlirContext ctx) {
     mlirRockTuningSetParam(tuningClone, tuningParam);
     if (mlirLogicalResultIsFailure(
             mlirPassManagerRunOnOp(applicabilityPm, tuningCloneOp))) {
-      // CHECK: Perfconfig "4,64,4,4,64,1,1,1" is not applicable
+      // CHECK-NOT: Perfconfig "4,64,4,4,64,1,1,1" is not applicable
```

Comment:
I think we can remove this line rather than having a CHECK-NOT; i suppose you are not getting any failures now ?

Reply:
Right, no failures now. I think this CHECK-NOT would ensure an early rejection?

Reply:
No, CHECK-NOT is for ensuring text doesn't appear.
I'd update this to `CHECK` for the first "not applicable" perf config in the tuning set.
But if there is none, you'll want to `CHECK-NOT: is not applicable`

Reply:
`CHECK-NOT: is not applicable` is better.